salt/client/mixins.py

```diff
@@ -382,6 +382,7 @@ def low(self, fun, low, print_event=True, full_return=False):
             data['pid'] = os.getpid()
             with salt.utils.files.fopen(jid_proc_file, 'w+b') as fp_:
                 fp_.write(serial.dumps(data))
+            del data['pid']
```

Comment:
I'm not quite following the relation between this and the rest of the PR. Help me out? :)

Reply:
@cachedout just addressing another incidental test failure. I suspect maybe the jenkins test matcher didn't include the runner tests on the original checkin of the proc handling stuff from Austin.

Reply:
Whew. I was afraid there was some crazy logic linking the grains change to this and I was starting to panic. ;)